1-tolean.py

```diff
@@ -607,9 +624,13 @@ def convert_to_lean_all():
         opts = parse_opt_file(f.read())
         for opt in opts:
             name, pre, src, tgt, ident_src, ident_tgt, used_src, used_tgt, skip_tgt = opt
+            while name in names:
+              name = name + "'"
+            names.append(name)
```

Comment:
```suggestion
```

Reply:
what's the difference? I honestly cannot tell 😅